app/src/main/java/com/likeminds/chatsampleapp/ChatMMApplication.kt

```diff
@@ -44,12 +46,18 @@ class ChatMMApplication : Application(), LMChatCoreCallback {
             application = this,
             lmChatCoreCallback = this,
             theme = chatTheme,
-            domain = "",
+            domain = deviceId(),
             enablePushNotifications = true,
-            deviceId = ""
+            deviceId = deviceId()
         )
     }
 
+    @SuppressLint("HardwareIds")
```

Comment:
Should we remove this?

Reply:
if we remove it, this is a warning, related to Hardware ids